apps/ui/src/components/EditorChoices.vue

```diff
@@ -3,6 +3,18 @@ import Draggable from 'vuedraggable';
 import { Draft } from '@/types';
 
 const proposal = defineModel<Draft>({ required: true });
+const choices: Ref<any[]> = ref([]);
+
+function handleAddChoice() {
+  proposal.value.choices.push('');
+  nextTick(() => choices.value[proposal.value.choices.length - 1].focus());
+}
+
+function handlePressEnter(index) {
+  if (!choices.value[index + 1]) return handleAddChoice();
+
+  nextTick(() => choices.value[index + 1].focus());
+}
```

Comment:
nit. we tend to put it in this order: `defineModel`/`defineProps`/`defineEmits` and then `refs`/`computed` and then functions. Now defineProps is at the bottom.